code/mob/living/critter/flockcritter_parent.dm

```diff
@@ -104,6 +104,11 @@
 	src.flock.removeDrone(src)
 	src.flock = null
 
+/mob/living/critter/flock/projCanHit(var/datum/projectile/P)
```

Comment:
```suggestion
/mob/living/critter/flock/projCanHit(datum/projectile/P)
```

Reply:
Also, since projCanHit() is just called by the parent Cross(), wouldn't it be slightly faster to put it in drone/Cross()?

Reply:
Well it technically would, but it seems to make sense to put the projectile hit code in `projCanHit`